src/box/lua/schema.lua

```diff
@@ -71,6 +71,18 @@ ffi.cdef[[
     box_txn_id();
     int
     box_txn_begin();
+    enum txn_isolation_level {
+        TXN_ISOLATION_DEFAULT,
+        TXN_ISOLATION_READ_COMMITTED,
+        TXN_ISOLATION_READ_CONFIRMED,
+        TXN_ISOLATION_BEST_EFFORT,
+        txn_isolation_level_MAX,
+    };
+    const char *txn_isolation_level_strs[txn_isolation_level_MAX];
+    int
+    box_txn_set_isolation_level(enum txn_isolation_level isolation_level);
+    int
+    box_txn_set_default_isolation_level(enum txn_isolation_level level);
```

Comment:
Not so sure this should be exposed via FFI: I think the code pattern for adding new options like in a76eb6ef is better.